docs/gettingstarted/private_send.md

```diff
@@ -43,7 +43,7 @@ nav_order: 4
   - If the send is unpinned:
     - No data is written to the blockchain at all
     - The message is marked confirmed immediately
-    - The sender does **not** get a `message_confirmed` event
+      - The sender receives a `message_confirmed` event immediately
```

Comment:
Was this supposed to be indented another level?

Reply:
It was intended, as it related to the confirmed...